README.md

```diff
@@ -1,161 +1,38 @@
-- Run seed-script
-- Run 'mock-all' endpoint
-
-### To "stage/demo" environment
-
-- Make sure to verify if the environment-settings are appropriately set on the
-  stage VM before publishing the release.
-- When a [release](#release-checklist) is published, it is automatically
-  deployed to the demo-server, via the webhook.
-- Wait until deploy is ready (by checking when the new version-number has appeared on the login-page of IBF-dashboard)
-- Note that the deployment logs can be followed in `/var/tmp/ibf-<yyyy-mm-dd>.stdout.log`
-- Run seed-script
-- Run 'mock-all' endpoint
+1. Install [NodeJS](https://nodejs.org/en/download)
+2. Install [Docker](https://docs.docker.com/get-docker)
+3. Download [IBF-system](https://github.com/rodekruis/IBF-system/archive/refs/heads/master.zip) source code
+4. Download and extract [raster-data.zip](https://510ibfsystem.blob.core.windows.net/rasters/raster-files.zip) into `services/API-service/geoserver-volume/raster-files`
+5. Setup env variables `cp example.env .env`
+6. Start API-service `npm run start:services`
+   - Open [https://localhost:3000/api](https://localhost:3000/api) in a web browser to check if the API-service is running
+   - Open [http://localhost:3000/docs](http://localhost:3000/docs) in a web browser to access the API-service documentation
+7. Start IBF-dashboard `npm run install:interface`
+   - Open [http://localhost:4200](http://localhost:4200) in a web browser to check if the IBF-dashboard is running
 
-### To "production" environment
+### Load data
 
-- SSH into the production server
-- Make sure to verify if the [environment variables](./example.env) are
-  appropriately set on the VM.
-- It might be needed to run the deploy-script as `azureuser` (same user with which the webhook runs the deploy-script on ibf-test and ibf-demo), by running `sudo su - azureuser` first. (As otherwise you would be running it with the personal user-account you logged in with.)
-- Manually run the [deploy script](./tools/deploy.sh) with the release which
-  should be deployed for the specific country, through `. ./tools/deploy.sh <release-id>`
-- Sometimes npm packages are not all automatically correctly installed. In case of issues with the api-service restart after deployment has finished, check the [this section](/docs/TROUBLESHOOT.md#install-packages-in-api-service) of the TROUBLESHOOT guide
+- Login via [/api/user/login](http://localhost:3000/docs#/--%20user%20--/UserController_login)
+- Load seed data via [/api/scripts/reset](http://localhost:3000/docs#/---%20mock%2Fseed%20data%20---/ScriptsController_resetDb)
+- Load mock data via [/api/scripts/mock](http://localhost:3000/docs#/---%20mock%2Fseed%20data%20---/ScriptsController_mock)
+- You can load data via the various [IBF-API](http://localhost:3000/docs) endpoints. Read [API for pipelines](https://github.com/rodekruis/IBF-system/wiki/API-for-pipelines) for details.
```

Comment:
PI-for-pipelines only refers to some of the endpoints that you could use to load data. And especially those endpoints are not typically meant to be used individually, but only as part of complete pipeline upload. You could include here also/alternatively static data endpoint and/or configuration update endpoints? There's a wiki on static data also.

Reply:
My draft had both links then I removed one coz I expect the user to find their way to the other. I can add it back if it's critical.

Reply:
not critical obviously.. never mind then